reference/toaztr.v1.yaml

```diff
@@ -350,6 +364,50 @@ components:
               type: array
               items:
                 $ref: '#/components/schemas/CaseResource'
+    PartnerBillResource:
+      title: PartnerBillResource
+      allOf:
+        - $ref: '#/components/schemas/Resource'
+        - type: object
+          properties:
+            attributes:
+              $ref: ../models/PartnerBill.v1.json
+      description: Représente une facture partenaire
+    PartnerBillResourceResponse:
+      title: PartnerBillResourceResponse
+      type: object
+      properties:
+        data:
+          $ref: '#/components/schemas/PartnerBillResource'
+    PartnerBillResourcePaginatedListResponse:
+      title: PartnerBillResourcePaginatedListResponse
+      type: object
+      allOf:
+        - $ref: '#/components/schemas/PaginatedResponse'
+        - type: object
+          properties:
+            data:
+              type: array
+              items:
+                $ref: '#/components/schemas/PartnerBillResource'
+    PartnerBillingParametersResource:
```

Comment:
Not sure I'm following why the parameters are resources here?
Is it done this way somewhere else?

Reply:
Indeed, it's not needed